djitellopy/tello.py

```diff
@@ -1088,12 +1088,16 @@ def frame(self):
         """
         Access the frame variable directly
         """
-        if self._with_queue:
+        if self.with_queue:
             return self.get_queued_frame()
 
-        with self._lock:
+        with self.lock:
             return self._frame
```

Comment:
here it is called self._frame (with underscore)

Reply:
Here the actual value of the frame (`self._frame`) is returned. If I were to use `self.frame`, then it would call the getter function of the frame again and it would get stuck in an infinite loop